README.md

```diff
@@ -1,81 +1,141 @@
-<div align="center">
+<br />
+<p align="center">
   <img src="https://user-images.githubusercontent.com/1626923/137092657-fb398d20-b592-4661-a1f9-4135db0b61d5.png" alt="Vue Storefront" height="80px" />
-</div>
-
+</p>
 
----------
+<h3 align="center">
+  Lightning-Fast Frontend Platform for Headless Commerce
+</h3>
+<p align="center">
+    Vue Storefront is the most popular and most advanced Frontend Platform for eCommerce.
+</p>
 
-# #TechForUkraine
-<div align="center">
-<p>
-       <img src="https://user-images.githubusercontent.com/1626923/155853691-d6d0a541-d3b9-40bf-b8f5-2d38303e9e49.png" />
+<p align="center">
```

Comment:
how about adding number of live stores below?

Reply:
Added :)